src/DwarfMap.hpp

```diff
@@ -0,0 +1,55 @@
+
+#ifndef SRC_DL_DWARFMAP_H_
+#define SRC_DL_DWARFMAP_H_
+
+#include <cstdlib>
+#include <map>
+#include <stdint.h>
+#include <libdwarf/libdwarf.h>
+#include <dwarf.h>
```

Comment:
This seems to only be compiling for me if I change this line to `#include <libdwarf/dwarf.h>`. Is this an error with the includes, or something else?